src/pool/cli.c

```diff
@@ -422,6 +422,13 @@ pool_connect_cp(tse_task_t *task, void *data)
 		D_GOTO(out, rc);
 	}
 
+	rc = dc_mgmt_pool_connect(pool);
```

Comment:
This naming scheme is a little confusing. Someone skimming through the code here might mistakenly assume that this function is about connecting to the pool. Suggest something more like `dc_mgmt_notify_pool_open()/dc_mgmt_notify_pool_close()`.

Reply:
renamed those calls and instead of notify_disconnect for the process exit call I called it notify_exit.